testserver/testserver.go

```diff
@@ -292,6 +301,13 @@ func NewTestServer(opts ...TestServerOpt) (TestServer, error) {
 		startCmd = "start"
 	}
 
+	var storeArg string
+	if serverArgs.storeInMem {
+		storeArg = "--store=type=mem"
```

Comment:
when using a `mem` store, you need to also specify a size: e.g. `--store=type=mem,size=0.2`

we could even consider making it so there's an option to pass in a whole store spec string, so there is more flexibility.